Gemfile

```diff
@@ -2,9 +2,10 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'sketch-in-ruby', github: 'meseker/sketch'
-gem 'units-in-ruby', github: 'meseker/units-ruby'
+gem 'sketch', github: 'aurorasolar/sketch', branch: 'misc/ruby-2.6.5'
+gem 'units-in-ruby', github: 'aurorasolar/units-ruby', branch: 'misc/ruby-2.6.5'
+gem 'geometry-in-ruby', github: 'aurorasolar/geometry', branch: 'misc/ruby-2.6.5'
```

Comment:
After merging all dependent gems' branches, we should update these back to master and update the revisions `bundle update --conservative sketch units-in-ruby geometry-in-ruby`